cl-base.obo

```diff
@@ -523,18 +523,6 @@ synonym: "mature eosinophil leukocyte" EXACT []
 synonym: "polymorphonuclear leucocyte" BROAD []
 synonym: "polymorphonuclear leukocyte" BROAD []
 is_a: CL:0000771 {is_inferred="true"} ! eosinophil
-intersection_of: CL:0000771 ! eosinophil
-intersection_of: BFO:0000051 GO:0098537
-intersection_of: BFO:0000051 PR:000007152
-intersection_of: BFO:0000051 PR:000007857
-intersection_of: BFO:0000051 PR:000007858
-intersection_of: BFO:0000051 PR:000014046
-intersection_of: BFO:0000051 PR:000014047
-intersection_of: BFO:0000051 PR:000022006
-intersection_of: RO:0002104 PR:000009129
-intersection_of: RO:0002215 GO:0002495
-intersection_of: RO:0002215 GO:0002540
-intersection_of: RO:0002215 GO:0045730
```

Comment:
Any idea where this was deleted? Was it intentional?

Reply:
not quite sure - definitely has something to do with [obsoleted lobed nucleus term](https://github.com/obophenotype/cell-ontology/issues/1130), but I would have thought that would change in [last release](https://github.com/obophenotype/cell-ontology/pull/1152) that @cmungall did ytd?

Reply:
![Screenshot 2021-06-29 at 11 43 28](https://user-images.githubusercontent.com/76212760/123784329-3a20f380-d8cf-11eb-9135-a583781f9aed.png)
Think this is it - was in cl.obo in the prerelease ytd

Reply:
I've also checked in protege and everything looks fine